Initial project files commit

jhsk-7 opened the pull request.

jhsk-7 committed:
Initial project files commit

jhsk-7 committed:
Update README.md

jhsk-7 committed:
Update README.md

jhsk-7 committed:
Modified routes. Started light and dark modes.

jhsk-7 committed:
Updated routing. Started dark and light modes.

jhsk-7 committed:
Delete MBW/mbw directory

jhsk-7 committed:
Update README.md

jhsk-7 committed:
Changes to Navbars and some routing.

jhsk-7 committed:
Test commit

jhsk-7 committed:
Merge branch 'new_project_files' of https://github.com/jhsk-7/Capstone into new_project_files

jhsk-7 committed:
added admin appointments pages

jhsk-7 committed:
added admin appointment id status patch buttons

jhsk-7 committed:
added react testing library

jhsk-7 committed:
Merge branch 'initial_project_files' into new_project_files

jhsk-7 committed:
Merge pull request #3 from jhsk-7/new_project_files

New project files

jhsk-7 committed:
started testing

jhsk-7 committed:
more testing files

jhsk-7 committed:
adding themes to pages

jhsk-7 committed:
light mode dark mode

jhsk-7 committed:
fixed navbar component

jhsk-7 committed:
fixed tests based on updated context

jhsk-7 committed:
updated for Render compatibility

jhsk-7 committed:
Fixing MaxListnersExceedWarning

jhsk-7 committed:
test comment

jhsk-7 committed:
testing render error message

jhsk-7 committed:
Patch

jhsk-7 committed:
Fix connection

jhsk-7 committed:
dynamic

jhsk-7 committed:
dynamic

jhsk-7 committed:
dynamic

jhsk-7 committed:
dynamic

jhsk-7 committed:
Render

jhsk-7 committed:
Render

jhsk-7 committed:
Render

jhsk-7 committed:
Render

jhsk-7 committed:
Render

jhsk-7 committed:
Render

jhsk-7 committed:
Render

jhsk-7 committed:
Making page shape more dynamic with page size

jhsk-7 committed:
Making side buttons dynamic

jhsk-7 committed:
Dynamic navbar settings

jhsk-7 committed:
fixing dynamic navbar settings

jhsk-7 committed:
fixing spelling

jhsk-7 committed:
fixing route spelling

jhsk-7 committed:
Enhance README with features and setup instructions

Expanded README with detailed user and admin features, tech stack, endpoints, and setup instructions.

jhsk-7 committed:
Remove duplicate sections in README.md

Removed duplicate entries in the README file.

namratabafna commented on mbw/package.json:
```diff
@@ -0,0 +1,38 @@
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
```
Missing install command

namratabafna commented on mbw/README.md:
````diff
@@ -0,0 +1,36 @@
+First, run the development server:
+
+```bash
+npm run dev
````
include npm install

namratabafna commented on mbw/src/app/admin/appointments/__tests__/adminappts.test.jsx:
```diff
@@ -0,0 +1,67 @@
+}));
+
+test('loads appointments and navigates to detail when a card is clicked', async () => {
+  axios.get.mockResolvedValueOnce({
```
can all tests move under single _tests?

namratabafna commented on mbw/src/app/admin/appointments/[id]/page.jsx:
```diff
@@ -0,0 +1,235 @@
+"use client";
+
+import { use, useEffect, useState } from "react";
```
is this unused import?

namratabafna commented on mbw/src/app/admin/appointments/[id]/page.jsx:
```diff
@@ -0,0 +1,235 @@
+          </div>
+
+          {/* Metadata */}
+          <div className={`text-xs mt-3 space-y-0.5 ${isDarkMode ? null : "text-gray-900"}`}>
```
use empty string instead of null. Here and in other places.

namratabafna commented on mbw/src/app/admin/appointments/page.jsx:
```diff
@@ -0,0 +1,154 @@
+        const { message} = normalizeError(err);
+        setError(message);
+      };
+      await findStatus();
```
Call normalizeError directly for simple and clear coding pattern.

namratabafna commented on mbw/src/app/admin/appointments/[id]/page.jsx:
```diff
@@ -0,0 +1,235 @@
+import { normalizeError } from "@/helpers/newErrorHandler";
+
+// Small date formatter to match your list pages
+function formatDate(input) {
```
Formatting is a bit off for this and other pages. Consider proper indentation before checkin

namratabafna commented on mbw/src/app/admin/appointments/[id]/page.jsx:
```diff
@@ -0,0 +1,235 @@
+          >
+            Completed
+          </button>
+          <button
```
Without type they default to "submit" inside forms. Add type="button". This and other places wherever necessary.

namratabafna replied:
From A11y perspective it does not mix roles when using narrator other accessibility assistive devices.

namratabafna reviewed (changes requested):
Left PR review comments, applies to other files as well.

jhsk-7 committed:
added more test to cover more branches

jhsk-7 committed:
Merge branch 'initial_project_files' of https://github.com/jhsk-7/Capstone into initial_project_files

jhsk-7 committed:
Enhance README with features and setup details

Expanded README with detailed user and admin features, tech stack, endpoints, and setup instructions.

jhsk-7 committed:
fix to preserve logout button after refresh
